guides/plugins/plugins/framework/flow/add-flow-builder-action.md

```diff
@@ -0,0 +1,360 @@
+
+![Choose a Flow Builder Action](../../../.gitbook/assets/flow-builder-trigger-action.png)
+
+
+You might need a modal to save your action config. For example , `swag-example-plugin-modal`.
+
+![Flow Builder create tag](../../../.gitbook/assets/flow-builder-tag.png)
+
+```tsx
+import template from './swag-example-plugin-modal.html.twig';
+const { Component } = Shopware;
+
+Component.register('swag-example-plugin-modal', {
+    template,
+
+    props: {
+        sequence: {
+            type: Object,
+            required: true,
+        },
+    },
+
+    data() {
+        return {
+            tags: [],
+        };
+    },
+
+    created() {
+        this.createdComponent();
+    },
+
+    methods: {
+        createdComponent() {
+            this.tags = this.sequence?.config?.tags || [];
+        },
+
+        onClose() {
+            this.$emit('modal-close');
+        },
+
+        onAddAction() {
+            const sequence = {
+                ...this.sequence,
+                config: {
+                    ...this.config,
+                    tags: this.tags
+                },
+            };
+
+            this.$emit('process-finish', sequence);
+        },
+    },
+});
+```
+
+```html
+{% block swag_example_plugin_modal %}
+    <sw-modal
+        class="swag-example-plugin-modal"
+        :title="$tc('swag-example-plugin.titleCreateTag')"
+        @modal-close="onClose"
+    >
+        {% block swag_example_plugin_modal_content %}
+            <sw-multi-tag-select
+                v-model="tags"
+                :label="$tc('swag-example-plugin.labelTags')"
+            />
+        {% endblock %}
+
+        {% block swag_example_plugin_modal_footer %}
+            <template #modal-footer>
+                {% block swag_example_plugin_modal_footer_cancel_button %}
+                    <sw-button
+                        class="swag-example-plugin-modal__cancel-button"
+                        size="small"
+                        @click="onClose"
+                    >
+                        {{ $tc('global.default.cancel') }}
+                    </sw-button>
+                {% endblock %}
+
+                {% block swag_example_plugin_modal_footer_save_button %}
+                    <sw-button
+                        class="swag-example-plugin-modal__save-button"
+                        variant="primary"
+                        size="small"
+                        @click="onAddAction"
+                    >
+                        {{ sequence.id
+                        ? $tc('swag-example-plugin.buttonSaveAction')
+                        : $tc('swag-example-plugin.buttonAddAction') }}
+                    </sw-button>
+                {% endblock %}
+            </template>
+        {% endblock %}
+    </sw-modal>
+{% endblock %}
+```
+
+### Demo
+Now you can view the demo for this custom Flow builder triger and action as below.
+
+![Flow Builder demo](../../../.gitbook/assets/flow-builder-demo.gif)
```

Comment:
The relative links are not correct I think - could you please double check these?

```
guides/plugins/plugins/framework/flow/add-flow-builder-action.md
../framework
../plugins

=> 
/guides/plugins/.gitbook/assets/...      
```